apertium-eng.eng.rlx

```diff
@@ -212,6 +217,54 @@ SELECT N IF (0 N) (0 Adj) (1 V) ;
 
 SELECT Adj IF (0 N) (0 Adj) (1 N) ;
 
+# Different cases for as ("as __ as")
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-3 Vbser) ;
+#Example (as):
+#You are as quick as I am.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-4 Vbser) ;
+#Example (as):
+#The Earth was formed as early as 4.54 billion years ago.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adj) (-3 Vbser) ;
```

Comment:
Isn't `-2` also `As`?

Reply:
You c an do `Adj` or `Adv` and not double these two rules...

Reply:
Okay, I've changed the rule to `SELECT Pr IF (0 As) (-2 As) (-1 Adv) (-1 Adj) (-3 Vbser) ;` to select both the adverb and the adjective.